pkgs/by-name/rd/rdesktop/package.nix

```diff
@@ -48,6 +48,12 @@ stdenv.mkDerivation (rec {
     "--disable-smartcard"
   ] ++ lib.optional (!enableCredssp) "--disable-credssp";
 
+  env = {
+    NIX_CFLAGS_COMPILE = toString [
+      "-Wno-error=incompatible-pointer-types"
```

Comment:
looks like a real bug that shouldn't be disabled: https://src.fedoraproject.org/rpms/rdesktop/blob/rawhide/f/53ba87dc174175e98332e22355ad8662c02880d6.patch